desktop/api/src/main/java/org/datacleaner/util/IconUtils.java

```diff
@@ -103,6 +103,7 @@
     public static final String MENU_OPEN = "images/menu/open.png";
     public static final String MENU_NEW = "images/menu/new.png";
     public static final String MENU_EXECUTE = "images/menu/execute.png";
+    public static final String MENU_REFRESH = "images/menu/refresh.png";
```

Comment:
Well, I know there is ACTION_REFRESH constant pointing to images/actions/refresh.png. First of all I did not want to use ACTION_* for menu item creation in code since there are other MENU_* constants. I guess I can create another constant with same path like MENU_REFRESH = images/actions/refresh.png which is also a bit strange for me. That is why I did the copy of the file itself and put it into different directory.